Added entry for HTML`<output>` tag (#745)
Codecademy/docs: content/html/concepts/tags/terms/output/output.md

---
Title: '<output>'

> Description: 'Represents the result of a calculation or user action.'

Reviewer: ```suggestion
Description: 'Displays the result of a calculation or user action.'
```

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Tags'
  - 'UI'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The `<output>` displays the result of a calculation or user action.
> 
> The `<output>` tag is typically used inside of a `<form>` element to dynamically show the result of user input, but it can also be used on its own.

Reviewer: ```suggestion
The `<output>` tag displays the result of a calculation or user action. It is commonly used inside of a [`<form>`](https://www.codecademy.com/resources/docs/html/tags/form) element to dynamically show the result of user input. However, the `<output>` tag can be used on its own.
```
